### PR TITLE
Add cancelInnerEvents property in ScrollView

### PR DESCRIPTION
**Pull request by pandamicro**
Re: cocos-creator/fireball#3953

Changes proposed in this pull request:
- Add cancelInnerEvents property in ScrollView

> Makes sure these boxes are checked before submitting your PR - thank you!
> - [x] If your pull request has gone "stale", you should **rebase** your work on top of the latest version of the upstream branch.
> - [x] If your commit history is full of small, unimportant commits (such as "fix pep8" or "update tests"), **squash** your commits down to a few, or one, discreet changesets before submitting a pull request.
> - To official teams:
>   - [x] Check that your javascript is following our [style guide](https://github.com/cocos-creator/fireball/blob/dev/.github/CONTRIBUTING.md) and end files with a newline
>   - [x] Document new code with comments in source code based on [API Docs](https://github.com/cocos-creator/fireball#api-docs)

@cocos-creator/engine-admins

### CODE REVIEW

**Comment by pandamicro** [comment]
@zilongshanren

File: cocos2d/core/components/CCScrollView.js [hunk 1]
```diff
@@ -272,12 +272,24 @@ var ScrollView = cc.Class({
 
         /**
          * !#en Scrollview events callback
-         * !#zh 滚动视图的事件回调函数
+         * !#zh 滚动过程中的事件回调函数
```
Comment (zilongshanren) [inline cocos2d/core/components/CCScrollView.js]: 这里为什么要修改。。 滚动结束也会调这个回调啊。。。

File: cocos2d/core/components/CCScrollView.js [hunk 2]
```diff
@@ -160,7 +160,7 @@ var ScrollView = cc.Class({
          * @property {Node} content
          */
         content: {
-            default: null,
+            default: undefined,
```
Comment (zilongshanren) [inline cocos2d/core/components/CCScrollView.js:163]: null 改成 undefined 有什么好处？

File: cocos2d/core/components/CCScrollView.js [hunk 3]
```diff
@@ -272,12 +272,24 @@ var ScrollView = cc.Class({
...
+         * !#zh 滚动过程中的事件回调函数
          * @property {Component.EventHandler[]} scrollEvents
          */
         scrollEvents: {
             default: [],
             type: cc.Component.EventHandler
+        },
+
+        /**
+         * !#en If cancel inner events is set to true, the scroll behavior will cancel touch events on inner content nodes of the scroll view
```
Comment (zilongshanren) [inline cocos2d/core/components/CCScrollView.js]: `If cancel inner events` 改成  `if cancelInnerEvents`? 这是一个属性，不能拆开吧？

File: cocos2d/core/components/CCScrollView.js [hunk 4]
```diff
@@ -272,12 +272,24 @@ var ScrollView = cc.Class({
...
+         * It's set to true by default.
+         * !#zh 如果这个属性被设置为 true，那么滚动行为会取消 ScrollView 的子节点上注册的触摸事件，默认被设置为 true。
+         * 注意，子节点上的 touchstart 事件仍然会触发，触点移动距离非常短的情况下 touchmove 和 touchend 也不会受影响。
+         * @property {Boolean} cancelInnerEvents
+         */
+        cancelInnerEvents: {
```
Comment (zilongshanren) [inline cocos2d/core/components/CCScrollView.js:290]: 这个manual 也要更新一下。

File: cocos2d/core/components/CCScrollView.js (hunk 2 above)
Comment (pandamicro) [inline cocos2d/core/components/CCScrollView.js:163]: undefined 不会被导出到序列化的 json 数据中

File: cocos2d/core/components/CCScrollView.js (hunk 4 above)
Comment (pandamicro) [inline cocos2d/core/components/CCScrollView.js:290]: 不明白什么意思

**Pull request merged**